scripts/utils/options.ts

```diff
@@ -0,0 +1,279 @@
+/**
+ * Use commander and prompts to gather a list of options for a script
+ */
+
+import prompts, { Falsy, PrevCaller, PromptType } from 'prompts';
+import type { PromptObject } from 'prompts';
+import program from 'commander';
+import type { Command } from 'commander';
```

Comment:
There seems to be some problems with importing types like this in commander, which is solved from commander v8, see this issue: https://github.com/tj/commander.js/issues/1465

To get the correct type in v6 you need to import it like this:
```suggestion
import * as commander from 'commander';
```

And then reference the type as `commander.Command`.
This will make sure you don't need the `any` casts in the test file.

Reply:
Or actually, you can use `program.Command` it seems as you already have `import program from 'commander';`